scripts/update_python_version.sh

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
+
+# This script is use to update the version of ITK used by ITKPythonPackage. It
+# is run nightly by a system with push permissions to the
+#
+#   git@github.com:InsightSoftwareConsortium/ITKPythonPackage.git
+#
+# repository.
+
+set -x -e
+
+workdir=/tmp/
+
+cd $workdir
+
+if test ! -d ITKNightlyMaster; then
+  git clone --branch nightly-master \
+    https://github.com/InsightSoftwareConsortium/ITK.git ITKNightlyMaster
+fi
+pushd ITKNightlyMaster/
+git checkout nightly-master
+git pull
+popd
+
+if test ! -d ITKPythonPackage; then
+  git clone --branch master \
+    git@github.com:InsightSoftwareConsortium/ITKPythonPackage.git ITKPythonPackage
+fi
+pushd ITKPythonPackage/
+git checkout master
+git pull
```

Comment:
Same thing. The advantage of reset hard is that it will not do merge commit in case there is a push force on the repo